mitmproxy/tools/web/app.py

```diff
@@ -324,6 +327,72 @@ def broadcast(cls, **kwargs):
 class ClientConnection(WebSocketEventBroadcaster):
     connections: ClassVar[set] = set()
 
+    def __init__(self, application: Application, request, **kwargs):
+        super().__init__(application, request, **kwargs)
+        self.application: Application = application
```

Comment:
Why are we assigning this again here? Can we just do a type annotation only?

Reply:
what do you mean precisely? somehow I need to access `self.application.master.view` below. 
I've specified again the type here: `self.application: Application = application`, because mypy was complaining.

Reply:
`super()` already assigns self.application. A type annotations [like this](https://github.com/mitmproxy/mitmproxy/blob/16660ca6e322df47c372afca7c5204fe91cd0005/mitmproxy/tools/web/app.py#L210) is enough.

Reply:
it works :) thanks for the advice!